packages/core/content-releases/shared/contracts/release-actions.ts

```diff
@@ -8,7 +8,7 @@ type ReleaseActionEntry = Entity & {
   // Entity attributes
   [key: string]: Attribute.Any;
 } & {
-  locale: string;
+  locale?: string;
```

Comment:
you can combine this and the & above, you're extending twice when you could do it once

Reply:
I had actually tried that before but I get this TS error:

```
Property 'locale' of type 'string' is not assignable to 'string' index type 'Any'.ts(2411)
```

I agree though, this isn't great and there's probably a better solution

Reply:
I could do this 🤔 

```
type ReleaseActionEntry = Entity & {
  // Entity attributes
  [key: string]: Attribute.Any | string;
  locale?: string;
};
```

Reply:
in that case I prefer the solution you currently have